styles.css

```diff
@@ -205,3 +222,102 @@ form {
   color: #42526e;
   padding-top: 10%;
 }
+
+@media (min-width: 992px) {
+  .flexnav {
+    justify-content: space-around;
+    padding: 5% 30%;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: 0%;
+    text-align: center;
+  }
+
+  .welcome {
+    display: none;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop {
+    display: flex;
+  }
+
+  .main {
+    background: url("./images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+  }
+
+  .landing-page {
+    background: transparent;
+    max-height: 42vh;
+    padding: 8% 16%;
+  }
+
+  .network {
+    padding: 0%;
+    justify-content: center;
+    column-gap: 20px;
+  }
+
+  .social {
+    flex: 0;
+  }
+
+  .works .works {
+    padding: 0%;
+  }
+
+  @supports (display: grid) {
+    .works .works {
+      grid-template-columns: 33% 33% 33%;
+    }
+  }
+
+  .aboutme {
+    background: url("./Images/illustration-about-me-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+    background-position: bottom;
+    padding: 0% 80px;
+  }
+
+  @supports (display: grid) {
+    .skills {
+      display: grid;
+      grid-template-columns: 31% 31% 31%;
+      justify-content: center;
+    }
+  }
+
+  .skills {
+    padding: 4% 8%;
+  }
+
+  #form {
+    justify-content: space-around;
+    align-items: center;
+    background: url("./images/illustration-form-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+    background-position: bottom;
+  }
+
+  @supports (display: grid) {
+    #form {
+      grid-template-columns: 40% 50%;
+    }
+  }
+
+  form .button {
+    align-self: flex-start;
+  }
+
+  .email {
+    display: none;
+  }
+}
```

Comment:
- [ ] [OPTIONAL] I have seen all your efforts right from the onset, I must say well done 👍🏽  for implementing all the recommendations by the previous reviewers, however, your background images are not showing up anymore, this may be as a result of your last commit(Your styling). Please fix this as it is important. See below:

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/149599998-bb42ff67-36f3-40ba-acef-2c0d189aed5d.png) | ![11](https://user-images.githubusercontent.com/58692590/149600013-30f1dd88-e471-4fc7-ace4-facfd6d29670.png)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![2](https://user-images.githubusercontent.com/58692590/149600400-e797f143-fdfb-42ef-9a42-d2881899ad4d.png) | ![22](https://user-images.githubusercontent.com/58692590/149600411-25576650-057e-4095-b30b-86fb566a136b.png)

Reply:
I'll look into it! Thank you!